docs/source/llms/tracing/index.rst

```diff
@@ -706,6 +725,16 @@ and the :py:meth:`mlflow.client.MlflowClient.delete_trace_tag` method to remove
     client.delete_trace_tag(request_id="12345678", key="tag_key")
 
 
+Async Logging
+-------------
+
+By default, MLflow Tracing are logged synchronously. This may introduce a performance overhead when logging Traces, especially when your MLflow Tracking Server is running on a remote server. If the performance overhead is a concern for you, you can enable **asynchronous logging** for tracing in MLflow 2.16.0 and later.
+
+To enable async logging for tracing, set the environment variable ``MLFLOW_ENABLE_ASYNC_LOGGING`` to ``true``. This will make the trace logging operation non-blocking and reduce the performance overhead.
```

Comment:
```suggestion
To enable async logging for tracing, set the environment variable ``MLFLOW_ENABLE_ASYNC_LOGGING`` to ``true``. This will make the trace logging operation non-blocking and reduce the performance overhead.
```

can we advise using `mlflow.config.enable_async_logging` instead?

Reply:
Makes sense!